### ISSUE TITLE
Obfuscation of data

### ISSUE THREAD

**Issue by shabana-ali** [hmrc/HMRC-content-style-guide#99]
I am an auditor on the Digital Inclusion, Accessibility and Standards (DIAS) team.

## Overview

There currently seems to be no guidance for obfuscation of data, for example, displaying account number when a user returns to a service.

We have seen services using asterisks `Account: **** 6789`.
Asterisks can cause confusion for some users, for example, the JAWS screen reader reads this as "star star star star 6789" and the NVDA screen reader says "four star 6789" which is not very clear.

Our recommendation is to use the wording `Account: ending in 6789`
This was received well by the UK Government Digital Slack content and UX community when mentioned as a better alternative for obfuscation by my colleague @adamliptrot-oc. Unfortunately this conversation is no longer available due to the message limit.

Tagging @fred-jackson to possibly contribute to this guidance. Thanks.

**Comment by Louisehmrc** [comment]
Hi @shabana-ali ,
I'm new to the design resources team, so just picking up this ticket. I know it's a bit old now, but have you gathered any further information or research since you posted this?

I will take it to our working group for discussion.

Best,
Louise

**Comment by shabana-ali** [comment]
Hi @Louisehmrc,

We've not seen any more examples of this during our accessibility audits but there was recently a query about [obfuscation on Slack](https://hmrcdigital.slack.com/archives/C04TW49AU/p1646753999099039). I'm tagging the content designer @nikkisugden  in case she provide user research on the matter.

Kind regards,
Shabana.

**Comment by shabana-ali** [comment]
HMRC pattern released https://design.tax.service.gov.uk/hmrc-design-patterns/hiding-information/
